application.windows64/java/Welcome.html

```
@@ -0,0 +1,28 @@
+<p>
+See the <a href="http://download.oracle.com/javase/7/docs/technotes/guides/plugin/">Java Plug-in</a> product
+documentation for more information on using the Java Plug-in product.
+<p> See the <a href=
+"http://www.oracle.com/technetwork/java/javase/overview/"
+>Java Platform</a> web site for 
+  more information on the Java Platform. 
+<hr>
+<font size="-2">
+Copyright (c) 2006, 2018, Oracle and/or its affiliates. All rights reserved.
+</font>
+<p>
```

HTMLHint (1.5.0)

[error] Tag must be paired, missing: [ </p></p></p></p></p> ], start tag match failed [ <p> ] on line 26.

(tag-pair)